### PR TITLE
allow land claim to place on south pole

### PR DESCRIPTION
**Pull request by bafolts**
This allows `LandClaim` to reserve the south pole at all times.

### CODE REVIEW

File: tests/utils/Vue.ts [hunk 1]
```diff
@@ -29,3 +30,18 @@ const Vue = require('vue');
 // setup for default exports
 Vue.default = Vue;
 
+// don't save to database during tests
+Database.getInstance = function () {
```
Comment (bafolts) [inline tests/utils/Vue.ts:34]: While also looking at how we serialise data and communicate with database I realised that during a unit test run the SQLite database is storing the test data! Adding a mock database here to avoid that overhead.

File: package.json [hunk 2]
```diff
@@ -40,18 +40,19 @@
     "extends": "@istanbuljs/nyc-config-typescript"
   },
   "scripts": {
+    "build": "npm run compile && npm run lint && npm run templates && npm run webpack",
     "clean": "rm -r ./build/ ./styles.css || true",
     "compile": "tsc",
-    "prep": "node make_static_json.js && lessc src/styles/common.less styles.css",
-    "build": "npm run prep && npm run compile && npm run lint && npm run templates && npm run webpack",
-    "start": "node build/server.js",
+    "cover": "nyc mocha --file build/tests/utils/Vue.js --recursive build/tests",
```
Comment (kberg) [inline package.json:46]: Why is this file change related to land claim?

File: src/cards/LandClaim.ts [hunk 3]
```diff
@@ -17,10 +20,19 @@ export class LandClaim implements IProjectCard {
     public canPlay(player: Player, game: Game): boolean {
       return game.board.getAvailableSpacesOnLand(player).length > 0;
```
Comment (kberg) [inline src/cards/LandClaim.ts:21]: Should this also call the new function?

File: src/cards/LandClaim.ts [hunk 4]
```diff
@@ -17,10 +20,19 @@ export class LandClaim implements IProjectCard {
...
     }
+    private getNonReservedLandSpaces(game: Game): Array<ISpace> {
+      return game.board.spaces.filter((space) => {
```
Comment (kberg) [inline src/cards/LandClaim.ts:24]: Under what circumstance is this algorithm only appropriate for this local use case? I wonder why this isn't centralized in Board.ts for example

File: package.json (hunk 2 above)
Comment (bafolts) [reply package.json:46]: I had linting errors while working on this solution so I added `fix` which is a script `gts` has which fixes linting errors. While in this file I put these in alphabetical order. They only changed order.

File: src/cards/LandClaim.ts (hunk 4 above)
Comment (bafolts) [reply src/cards/LandClaim.ts]: I moved this to `Board.ts`. Since `HellasBoard` always calls to filter `SpaceName.HELLAS_OCEAN_TILE` this separate function made the most sense to me. Land claim is a fairly unique project card. Moving the function to `Board` lessens the number of imports `LandClaim` has to take on though.

File: tests/utils/Vue.ts (hunk 1 above)
Comment (kberg) [reply tests/utils/Vue.ts]: 😂😂😂😆😁😄😃🙂🤔😐🙁😦😦😢😢😭😭😭

**Review by kberg — APPROVED** [review]

**Comment by bafolts** [comment]
Randomly placed tiles for single player game are sometimes place a tile on `SpaceName.HELLAS_OCEAN_TILE` and the test is failing, putting up fix.
